Public/Src/FrontEnd/Script/Ambients/Transformers/AmbientTransformer.Process.cs

```diff
@@ -1179,6 +1181,17 @@ private void ProcessUnsafeOptions(Context context, ProcessBuilder processBuilder
             if (Converter.ExtractOptionalBoolean(unsafeOptionsObjLit, m_unsafeAllowPreservedOutputs) == true)
             {
                 processBuilder.Options |= Process.Options.AllowPreserveOutputs;
+
+                if (context.FrontEndHost.Configuration.Sandbox.PreserveOutputsForIncrementalTool)
+                {
+                    processBuilder.Options |= Process.Options.IncrementalTool;
+                }
+            }
+
+            // UnsafeExecuteArguments.incrementalTool
+            if (Converter.ExtractOptionalBoolean(unsafeOptionsObjLit, m_unsafeIncrementalTool) == true)

```

Comment:
Don't forget to add the option to the type in the dsc file at:

[https://github.com/microsoft/BuildXL/blob/master/Public/Sdk/Public/Transformers/Transformer.Execute.dsc#L216](https://github.com/microsoft/BuildXL/blob/master/Public/Sdk/Public/Transformers/Transformer.Execute.dsc#L216)